main.go

```diff
@@ -134,12 +134,10 @@ func main() {
 	} else {
 		mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 			Scheme:                 scheme,
-			MetricsBindAddress:     metricsAddr,
-			Port:                   9443,
+			Metrics:                metricsserver.Options{BindAddress: metricsAddr},
 			HealthProbeBindAddress: probeAddr,
 			LeaderElection:         enableLeaderElection,
 			LeaderElectionID:       "28185cee.integreatly.org",
-			Namespace:              watchNamespace,
```

Comment:
are you sure this is correct? 
To me, this looks like we are making RHOAM cluster scoped

Reply:
Yes, it's one of main points, I planned to recheck and update it after weekend.
Given that sigs.k8s.io/controller-runtime v0.17.2 does not directly support namespace scoping through the ctrl.Options struct, we need to implement namespace scoping within the controller setup.
It's done now, but need test it - https://github.com/valerymo/integreatly-operator/blob/MGDAPI-6289/main.go#L148   , https://github.com/valerymo/integreatly-operator/blob/MGDAPI-6289/controllers/rhmi/rhmi_controller.go#L1168-L1170
Thank you !

Reply:
implementation changed, to be same as in 3scale and CRO

Reply:
Further updates done, now it looks all good. Thanks Valery